### DISCUSSION TITLE
Just a quick question: what is the benefit i will get from this lib?

### DISCUSSION THREAD

**Discussion by jcyuan** [i18next/next-i18next#2299] [Q&A] [answered]
Hi guys, i wonder what is the benefit if i use next-i18next? i'm currently develping with next.js14 and it seems there is a built-in i18n module in it but just i'm not familiar with it, so here i got this question.
thank you~~

**Comment by adrai** [comment, marked as answer]
There is no built-in module in Next.js for proper i18n...... for the pages setup, there is i18n routing... and next-i18next will do the rest...
for the app directory setup, there is nothing... and you can use this: https://locize.com/blog/next-app-dir-i18n/

**Reply by jcyuan** [reply]
I got it, thank you very much.  :)
